react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: 0,
+      image: '../dist/images/001.png',
```

Comment:
You don't have to store both of these. your `counter` should be enough, because you can use it as an index into your `props.data` when you want the image path.